graphql/generated/generated.go

```diff
@@ -3945,6 +3954,40 @@ func (ec *executionContext) _Stats_ethRPCRateLimitExpiredRequests(ctx context.Co
 	return ec.marshalNInt2int(ctx, field.Selections, res)
 }
 
+func (ec *executionContext) _Stats_secondaryRendezvous(ctx context.Context, field graphql.CollectedField, obj *gqltypes.Stats) (ret graphql.Marshaler) {
+	defer func() {
+		if r := recover(); r != nil {
+			ec.Error(ctx, ec.Recover(ctx, r))
+			ret = graphql.Null
+		}
+	}()
+	fc := &graphql.FieldContext{
+		Object:   "Stats",
+		Field:    field,
+		Args:     nil,
+		IsMethod: false,
+	}
+
+	ctx = graphql.WithFieldContext(ctx, fc)
+	resTmp, err := ec.ResolverMiddleware(ctx, func(rctx context.Context) (interface{}, error) {
+		ctx = rctx // use context from middleware stack in children
+		return obj.SecondaryRendezvous, nil
+	})
+	if err != nil {
+		ec.Error(ctx, err)
+		return graphql.Null
+	}
+	if resTmp == nil {
+		if !graphql.HasFieldError(ctx, fc) {
+			ec.Errorf(ctx, "must not be null")
+		}
+		return graphql.Null
+	}
+	res := resTmp.([]string)
+	fc.Result = res
+	return ec.marshalNString2ᚕstringᚄ(ctx, field.Selections, res)
```

Comment:
Strange unicode characters here, but the same happens in some other functions, so not caused by this PR. Still, what's going on here?

Reply:
It's part of graphql generation. I'm not completely sure what these replacements mean and why unicode is being used, but I did find an instance where they are using these unicode characters during codegen: https://github.com/99designs/gqlgen/blob/f4669ba9b54cdda6340c6e0c16d05eff2ee4fa21/codegen/templates/templates.go#L232.